server/sonar-server/src/main/java/org/sonar/server/rule/RegisterRules.java

```diff
@@ -21,13 +21,16 @@
 
 import com.google.common.collect.Iterables;
 import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import java.util.Set;
+import java.util.function.Function;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.function.Function'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Feh%2FSONAR-10311&issues=AWFRf5ZjsS4rYBZqFGC6&open=AWFRf5ZjsS4rYBZqFGC6)

[ISSUE_KEY]: AWFRf5ZjsS4rYBZqFGC6